pkg/service/backup_routine_handler.go

```diff
@@ -158,7 +172,7 @@ func (h *BackupRoutineHandler) runFullBackupInternal(ctx context.Context, now ti
 		return err
 	}
 
-	h.lastRun.full = now
+	h.lastRun.Full = &now
```

Comment:
It is not a new thing, but it is better to synchronize writes and reads to prevent data races.

Reply:
No backup of the same type for the same routine will run simultaneously, no need for lock

Reply:
There are two jobs scheduled for the full and incremental backup that may concurrently access those variables.